config.test.yaml

```diff
@@ -159,6 +163,83 @@ templates:
                   options:
                     host: http://appservice.wmflabs.org
 
+      /{module:summary}:
```

Comment:
Why is this here? Looks like this is doing the same thing as `mods/restbase/summary_v1` so why is it needed?